core/variant/variant_call.cpp

```diff
@@ -1126,10 +1126,6 @@ static void _register_variant_builtin_methods() {
 	bind_method(Quat, cubic_slerp, sarray("b", "pre_a", "post_b", "weight"), varray());
 	bind_method(Quat, get_euler, sarray(), varray());
 
-	// FIXME: Quat is atomic, this should be done via construcror
```

Comment:
@akien-mga This comment indicates that the goals of this PR are [reduz approved(tm)](https://github.com/godotengine/godot/blame/89f605c717e271050a6cc61547ea3d25996a19c4/core/variant_call.cpp#L1603).

(this is not a suggested change, it's just me pointing something out)